<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertOscilloscopeFileForm</class>
 <widget class="QWidget" name="ConvertOscilloscopeFileForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert oscilloscope file</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save points every:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="file_path_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_file_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convert_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="timeMeasureGroupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Time measure:</string>
   </property>
   <widget class="QRadioButton" name="secondsRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="minutesRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>minutes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hoursRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>hours</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="daysRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>days</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="normalize_checkBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>93</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Normalize</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
